src/content-helper/editor-sidebar/editor-sidebar.scss

```diff
@@ -186,6 +186,365 @@
 	}
 }
 
+.wp-parsely-content-helper .wp-parsely-performance-panel {
+	padding: calc(16px);
+	border-top: 1px solid rgb(221, 221, 221);
+	margin-top: -1px;
+
+	.panel-body {
+		width: 100%;
+
+		/** View in Parse.ly button */
+		.components-button.wp-parsely-view-post {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			width: 100%;
+		}
+	}
+
+	.performance-stat-panel {
+		display: flex;
+		padding: to_rem(6px) 0 to_rem(16px) 0;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: 8px;
+		align-self: stretch;
+
+		.panel-header {
+			display: flex;
+			height: to_rem(16px);
+			width: 100%;
+			align-items: center;
+			gap: 6px;
+			margin-bottom: 0;
+
+			/** Level 2 heading */
+			&.level-2 {
+				margin-bottom: to_rem(15px);
+
+				h2 {
+					margin-bottom: 0;
+				}
+			}
+
+			&.level-3 {
+
+				/** Level 3 heading */
+				h3 {
+					margin-bottom: 0;
+					line-height: to_rem(16px);
+					font-family: "SF Pro Text", sans-serif;
+					font-size: 11px;
+					font-style: normal;
+					font-weight: 600;
+					text-transform: uppercase;
+				}
+			}
+		}
+
+		.panel-subtitle {
+			display: flex;
+			flex: 1 0 0;
+			height: 16px;
+			align-items: center;
+			gap: 6px;
+			align-self: stretch;
+			color: var(--Gutenberg-Gray-700, #757575);
+
+			/* Label */
+			font-family: "SF Pro Text", sans-serif;
```

Comment:
It came straight from FIgma - I used the styles provided there with some tweaks. But I'm glad you asked, because that's something I wanted to discuss, since I'm not sure if we should be using this font, and the fall back `sans-serif` I picked was really just temporarily.

Reply:
@dabowman?

Reply:
As we discussed, I removed the `font-famliy` rules that were introduced to use the default wp-admin fonts. 

As a side-note, on that same commit, I also moved all the Performance Stats rules to its own file in the `performance-stats` directory, to help uncluttering the `editor-sidebar.scss` file and hopefully make it more maintainable going forward.